File: code-coaches/medium/tax-free/tax_free.py
def get_total(prices: list[int]) -> float:
    tax = 1.07
    total = sum(price * tax if price < 20 else price for price in prices)
    total_rounded = round(total * 100) / 100
    return total_rounded


def main():
    prices = list(map(int, input().split(",")))
    print(get_total(prices))


if __name__ == "__main__":
    main()

File: code-coaches/medium/tax-free/tax_free.rb
def get_total(prices)
  tax = 1.07
  total = prices.sum { |price| price < 20 ? price * tax : price }
  total_rounded = (total * 100).round / 100.0
end

def main
  prices = gets.chomp.split(",").map(&:to_i)
  puts get_total(prices)
end

if __FILE__ == $0
  main
end
